src/jit/codegenxarch.cpp

```diff
@@ -6474,7 +6474,7 @@ void CodeGen::genJmpMethod(GenTreePtr jmp)
             }
         }
 
-#if FEATURE_VARARG
+#if FEATURE_VARARG && defined(_TARGET_AMD64_)
```

Comment:
As discussed offline, we need to review whether the logic in lines 6380-6386 is applicable to x86.